source/table_view.tsx

```diff
@@ -1,32 +1,69 @@
 import * as React from 'react';
-import * as ReactDOM from 'react-dom';
 import { TableModel } from './table_model';
 
 interface Properties {
 
   /** The model to display. */
   model: TableModel;
+
+  /**  The label for the columns of the table.*/
+  header?: string[]
+
+  /** Specifies the CSS class. */
+  className?: string;
+
+  /** The CSS style to apply. */
+  style?: any;
 }
 
 /** Renders a TableModel to HTML. */
 export class TableView extends React.Component<Properties> {
+  public static readonly defaultProps = {
+    header: [] as string[],
+    style: {}
+  };
+
   public render(): JSX.Element {
+    const header = [];
+    for(let i = 0; i < this.props.header.length; ++i) {
+      header.push(
+        <th style={this.props.style.th}
+            className={this.props.className}>
```

Comment:
You will need to give a unique key or else this will warn you and there will be issues when updating.